src/test/java/org/folio/circulation/services/FeeFineFacadeTest.java

```diff
@@ -126,15 +128,20 @@ public void shouldForwardFailureIfAnAccountIsNotRefunded() throws Exception {
       .withLoan(Loan.from(new JsonObject().put("id", UUID.randomUUID().toString())));
   }
 
-  private RefundAccountCommand refundCommand() {
+  private AccountRefundCancelCommand refundCommand() {
     final JsonObject account = new JsonObject()
       .put("feeFineType", "Lost item fee")
       .put("amount", 50.0)
-      .put("remaining", 50.0)
+      .put("remaining", 0.0)
       .put("id", UUID.randomUUID().toString());
```

Comment:
Is it no longer possible to partially refund an account?

Reply:
It is possible. The paid/transferred amount is refunded and rest is cancelled. But for this test I have to tests error forwarding, so no needs to prepare more complicated case.

Reply:
Ok, thanks for explaining that